src/ray/gcs/gcs_server/gcs_actor_manager.h

```diff
@@ -45,6 +45,14 @@ class GcsActor {
   explicit GcsActor(rpc::ActorTableData actor_table_data)
       : actor_table_data_(std::move(actor_table_data)) {}
 
+  /// Create a GcsActor by actor_table_data and task_spec.
+  ///
```

Comment:
Can you update the docstring that this constructor is only for alive actors?

Reply:
ok